blocks/v2-accordion-column/v2-accordion-column.css

```diff
@@ -0,0 +1,194 @@
+.section > .v2-accordion-column-wrapper.full-width {
+  padding: 0 16px;
+}
+
+.v2-accordion-column__header-container h2 {
+  font-family: var(--ff-headline-medium);
+}
+
+.v2-accordion-column__item-header-button {
+  background: transparent;
+  border: 0;
+  color: var(--c-primary-black);
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0 0 8px;
+  width: 100%;
+}
+
+.v2-accordion-column__item-header-button:hover,
+.v2-accordion-column__item-header-button:focus {
+  background: transparent;
+  cursor: pointer;
+}
+
+.v2-accordion-column__item-title {
+  font-family: var(--ff-subheadings-medium);
+  font-size: var(--headline-1-font-size);
+  letter-spacing: var(--headline-1-letter-spacing);
+  line-height: var(--headline-1-line-height);
+  margin: 0;
+}
+
+.v2-accordion-column__close {
+  transform: rotate(180deg);
+  transition: transform var(--duration-small) var(--easing-standard);
+}
+
+.v2-accordion-column__item .icon svg {
+  height: 24px;
+  width: 24px;
+}
+
+.v2-accordion-column__item .icon svg,
+.v2-accordion-column__item a .icon svg {
+  display: flex;
+}
+
+.v2-accordion-column__item a .icon svg {
+  height: 16px;
+  width: 16px;
+  stroke: currentcolor;
+}
+
+.v2-accordion-column__item.active .v2-accordion-column__close {
+  transform: rotate(0);
+}
+
+.v2-accordion-column__item :is(
+  .v2-accordion-column__item-image,
+  .v2-accordion-column__item-description){
+  display: none;
+}
+
+.v2-accordion-column__item {
+  margin: 0 0 16px;
+  border-bottom: 2px solid var(--c-primary-black);
```

Comment:
should be `border-bottom: 1px solid var(--Secondary-Steel-A7ABAF, #A7ABAF);`

Reply:
I used our own name, not figma one